packages/dd-trace/src/telemetry/logs/log-collector.js

```diff
@@ -73,8 +73,10 @@ const logCollector = {
       }
       const hash = createHash(logEntry)
       if (!logs.has(hash)) {
-        logs.set(hash, logEntry)
+        logs.set(hash, errorCopy(logEntry))
```

Comment:
For the record: `errorCopy` function is OK but it is not needed :)

`logCollector` expects a `logEntry` not an `Error`.

 `logEntry` is an object that meets telemetry logs payload schema: `message`, `level`, `count`, `tags`, `stack_trace` and `tracer_time` fields. So you could simply add here `logEntry.count = 1` instead calling `errorCopy(logEntry)`

Reply:
Ah! I had done it that way to prevent issues where a userland error would then get mutated to have the .count property. Is that something that can never happen?